core/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -76,19 +75,17 @@
  * between the lucene index an the transaction log file. This UUID is used to prevent accidental recovery from a transaction log that belongs to a
  * different engine.
  * <p>
- * Each Translog has only one translog file open at any time referenced by a translog generation ID. This ID is written to a <tt>translog.ckp</tt> file that is designed
- * to fit in a single disk block such that a write of the file is atomic. The checkpoint file is written on each fsync operation of the translog and records the number of operations
- * written, the current translogs file generation and it's fsynced offset in bytes.
+ * Each Translog has only one translog file open for writes at any time referenced by a translog generation ID. This ID is written to a
+ * <tt>translog.ckp</tt> file that is designed to fit in a single disk block such that a write of the file is atomic. The checkpoint file
+ * is written on each fsync operation of the translog and records the number of operations written, the current translogs file generation
+ * and it's fsynced offset in bytes.
```

Comment:
Since we are here editing these already, and I should updated this when adding these to the checkpoint, the Javadoc here is inaccurate since we also write the minimum and maximum sequence number in the corresponding translog, and the global checkpoint.